README.md

```
@@ -1,204 +1,300 @@
 
-You can integrate Amazon Lex Bot with Kommunicate by following the below steps. [Here](https://www.youtube.com/watch?v=cuaAiutVkow) is a video for the same.
+<uses-permission
+    android:name="android.permission.ACCESS_FINE_LOCATION"
+    tools:node="merge" />
+```
 
-### Step 1: Get your credentials from Amazon Lex
+![Permissions for location](images/permissions-location.png)
 
-**A. Access Key ID & Secret access key:** An access key ID and secret access key are required to sign requests you make using the AWS SDKs. To get your access key [sign into your AWS console](https://aws.amazon.com/console/) as a Root or IAM user.
+For more information on authentication, push notification, customization, etc, check out our official documentation [here](https://docs.kommunicate.io/docs/android-installation).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)